fineract-provider/src/main/java/org/apache/fineract/organisation/monetary/domain/NumberToWordsConverter.java

```diff
@@ -0,0 +1,265 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.fineract.organisation.monetary.domain;
+
+import java.text.DecimalFormat;
+
+public class NumberToWordsConverter {
+    public enum Language {
+        ENGLISH, SPANISH
+    }
+
+    public static String convertToWords(int number, Language language) {
+        switch (language) {
+            case SPANISH:
+                return SpanishConverter.convertToWords(number);
+            default:
+                return EnglishConverter.convertToWords(number);
+        }
+    }
```

Comment:
So number with decimal places can be rounded/stripped before the conversion,
It might be tricky to convert decimal places to words